notebooks/mock-notebooks/login/User Registration.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>Should/Can we specify when the domain is offline?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think we can do that. Will add a case to showcase that.</p>